### ISSUE TITLE
Configuration Field Rename Request

### ISSUE THREAD

**Issue by Invadermonky** [Un-Original1/BeastSlayer#27]
tldr; Revert the renaming of the [`sandmonsterSpawnRate`](https://github.com/Un-Original1/BeastSlayer/blob/d83efeff48ee7a3b632fb668255bd82456aa2d39/src/main/java/com/unoriginal/beastslayer/config/BeastSlayerConfig.java#L39) configuration field to `sandmonsterSpawnChance`.

During the most recent update the `sandmonsterSpawnChance` field in the configuration was changed to `sandmonsterSpawnRate`, breaking compat with [Just Enough Magiculture](https://www.curseforge.com/minecraft/mc-mods/just-enough-magiculture).

As this config option name appears to be non-standard to the rest of the configuration options which use the `<monstername>SpawnChance` syntax, would it be possible to change the field name back to the original?

**Comment by Un-Original1** [comment]
Whoops, seems I named the random spawn fail chance (which I removed last update) and the spawn weight wrong, I'll update it asap, thanks for pointing it out.

**Comment by Un-Original1** [comment]
It has been reverted
